### DISCUSSION TITLE
Simple way of setting labels

### DISCUSSION THREAD

**Discussion by Faiiya** [JordanKnott/taskcafe#99] [Ideas]
Hello.
on some kind of backlog app i remember the ability to set up tags just by hovering over the card and pressing a number or "+".

so you got to a card, hover and then press "1" and you set up your first label, and with "+" it just keep iterating over your labels.

is this possible?

**Comment by JordanKnott** [comment]
The ability to manage labels along with other attributes through keyboard shortcuts is planned!
The current plan is that you will be able to open the label manager while hovering over a task by pressing L. Pressing the 1-0 keys while hovering over a task would just add the associated label 
 I do like your suggestion of being able to press + to iterate over other labels though
